pkg/check/pushsync/check_lightnode.go

```diff
@@ -0,0 +1,95 @@
+package pushsync
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/ethersphere/beekeeper/pkg/bee"
+	"github.com/ethersphere/beekeeper/pkg/beeclient/api"
+	"github.com/ethersphere/beekeeper/pkg/random"
+)
+
+// CheckChunks uploads given chunks on cluster and checks pushsync ability of the cluster
+func CheckLightChunks(c *bee.Cluster, o Options) error {
+	ctx := context.Background()
+	rnds := random.PseudoGenerators(o.Seed, o.UploadNodeCount)
+	fmt.Printf("seed: %d\n", o.Seed)
+
+	overlays, err := c.FlattenOverlays(ctx, "bee", "bootnode")
+	if err != nil {
+		return err
+	}
+
+	drones := c.NodeGroup("drone")
```

Comment:
can we call drones lightnodes?

Reply:
I can make the code change, but I don't know if some other scripts have to be updated? @svetomir

Reply:
No, it is enough to do a code change, but I would pick a different name. Lightnode is very specific, and that node group can be anything.